### ISSUE TITLE
Deleting an annotation has broken keyboard interaction

### ISSUE THREAD

**Issue by avaaec** [ManifoldScholar/manifold#2246]
## Steps to reproduce
1. Go to reader
2. Make an annotation
3. Go to Notes panel
4. Select annotation in list
5. Use the keyboard to focus the delete option for that annotation
6. Activate the delete button with enter or space
7. Notice that keyboard focus jumps to the close panel button
8. Navigate back to the delete button that has now become disabled
9. Notice that there is no semantic or hierarchical difference between the new confirm and cancel buttons and the other actions for the annotation like edit and reply

## Observed Behavior
When the delete button is activated focus jumps to the "close panel" button, the delete button becomes keyboard inaccessible and two new buttons, "confirm" and "cancel" appear. The loss of focus is a clear problem for keyboard users. Additionally the confirm and cancel buttons are un-intuitive and confusing for non visual users. Visually they are grouped by proximity to the disabled delete button but they are not grouped in any ways that help non visual computer users. A screen reader user for example sees confirm and cancel as being on the same hierarchical level as reply, edit, and flag. For example browsing through the list of actions after activating the delete button I see:
```
List with 1 item
    navigation landmark
         list with 4 items
            Reply, Edit, Confirm, Delete, Flag
```
This is extremely confusing because
1. The entire list is inside a single item list (I think the feedback that lists should not be used for styling has been tracked elsewhere)
2. There is a <nav> where there should be a <span> causing assistive tech to see a navigation landmark (https://www.w3.org/WAI/GL/wiki/Using_ARIA_landmarks_to_identify_regions_of_a_page) 
3. There are five items in a list of four items!?

## Expected Behavior
When the delete button is activated the users focus should go to "confirm". There should be a hierarchical difference between the confirm and cancel buttons and the other action controls for the annotation, a possible solution is to put these two buttons in a list. Additionally there should be some semantic link between the confirm and cancel buttons and their purpose. This could happen through a specific aria describe by (i.e. "confirm delete button") or through being in a list titled delete (i.e. "delete list with 2 items"). The non visual experience for these controls should look more like this: (line breaks added for clarity)
```
List of 4 items
    Reply
    Edit
    List of two items named delete
         Confirm, Cancel
    Flag
```

## Test Environment 
NVDA 2019.1.1
ESR Firefox 60.7.0esr

---
Let me know if you have any complaints / requests for how I am reporting bugs. :)

**Comment by zdavis** [comment]
We couldn't ask for a better bug report than this. Thank you!

**Comment by dananjohnson** [comment]
Agreed! Thanks @avaaec for these super helpful and thorough bug reports!

**Comment by avaaec** [comment]
Fix verified, however ideally the users focus would be taken back to their previous location after they delete the last annotation for a particular snippet of text. For example if I go from the notes panel to a specific annotation and then delete that annotation I would expect my focus to be taken back to the notes panel. Currently my focus is returned to the navigation banner. This is a low priority item but is worth considering.
